master.sh

```diff
@@ -40,13 +40,7 @@ init() {
 	fi
 
 	# Set working mode
-	if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
-		echo "Start k8s cluster in installation mode"
 		RESTART_POLICY="always"
```

Comment:
OK, this sounds reasonable. My question was rather: in the `install` case, should I replace the existing `hyperkube + restart_policy=always` with the new `kubelet as service` installation, or still keep them both for some reason? I don't see the benefit of having both right now, maybe you could clarify this.